packages/bits/src/lib/sorter/sorter-key-control.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable, QueryList } from "@angular/core";
+import { ListKeyManager, LiveAnnouncer } from "@angular/cdk/a11y";
+import isNull from "lodash/isNull";
+
+import { MenuItemBaseComponent } from "../menu";
+import { OverlayComponent } from "../overlay/overlay-component/overlay.component";
+import { KEYBOARD_CODE } from "../../constants";
+import { ANNOUNCER_CLOSE_SORTER_LIST_MESSAGE, ANNOUNCER_OPEN_SORTER_LIST_MESSAGE_SUFFIX } from "./constants";
+
+@Injectable()
+export class SorterKeyControlService {
```

Comment:
Maybe this to match the [ToolbarKeyboardService](https://github.com/solarwinds/nova/blob/7d0224b66cb4dc62d68ead5e21fdff51820d221c/packages/bits/src/lib/toolbar/toolbar-keyboard.service.ts) name:

```suggestion
export class SorterKeyboardService {
```